### PR TITLE
GH-93 Updates jacoco maven plugin version to 0.8.1

### LINKED ISSUE #93: Update jacoco plugin version
As a developer
I want to update jacoco maven plugin version
So that it does not enforce coverage for empty private constructors

See https://github.com/jacoco/jacoco/pull/529

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Fixes #93

### CODE REVIEW

File: tracer-core/src/test/java/org/zalando/tracer/EnforceCoverageTest.java [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-    @Test
-    void shouldUseTraceListenersConstructor() {
-        new TraceListeners();
-    }
-
-
-    @Test
-    void shouldCoverTracerBuilderToString() {
-        assertThat(Tracer.builder(), hasToString(notNullValue()));
-    }
```
Comment (whiskeysierra) [inline tracer-core/src/test/java/org/zalando/tracer/EnforceCoverageTest.java]: Not a constructor test

Comment (AlexanderYastrebov) [reply tracer-core/src/test/java/org/zalando/tracer/EnforceCoverageTest.java]: No it is not, and it also looks useless

Comment (AlexanderYastrebov) [reply tracer-core/src/test/java/org/zalando/tracer/EnforceCoverageTest.java]: I.e. removal look like an improvement :)

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**